lib/vcloud/launcher/vm_orchestrator.rb

```diff
@@ -23,7 +23,7 @@ def customize(vm_config)
 
         preamble = vm_config[:bootstrap] ? generate_preamble(vm_config) : ''
 
-        @vm.configure_guest_customization_section(preamble)
+        @vm.configure_guest_customization_section(preamble) if (!preamble.nil? && !preamble.empty?)
```

Comment:
What happens when the preamble value is literally set to `nil`? Can a test be added for that?
It's an edge case, but it's possible that it could occur.